dashboard/dashboard_functions.py

```diff
@@ -2,9 +2,35 @@
 import pandas as pd
 import altair as alt
 import streamlit as st
+from PIL import Image
 
 
-def make_date_radio():
+from download_screenshot import download_data_file, get_s3_client
+
+BUCKET = 'c9-internet-archiver-bucket'
+
+# TODO Add most popular site with screenshot, Add proper sorting to grouped bar charts
+
+
+def make_metrics(scrape_data: pd.DataFrame, interaction_data: pd.DataFrame) -> None:
+    archives = scrape_data[scrape_data["is_human"] == True]['url'].count()
+    visits = interaction_data[interaction_data["type"]
+                              == 'visit']['url'].count()
+    saves = interaction_data[interaction_data["type"]
+                             == 'save']['url'].count()
+
+    col1, col2, col3 = st.columns(3)
+    with col1:
+        st.metric("Archives", archives)
+
+    with col2:
+        st.metric("Visits", visits)
+
+    with col3:
+        st.metric("Saves", saves)
```

Comment:
really good to have all of these separate, outlines why people are using the site